swarm/pss/client/client_test.go

```diff
@@ -160,6 +161,15 @@ func TestClientHandshake(t *testing.T) {
 		log.Warn("ok", "idx", i, "got", got)
 		time.Sleep(time.Second)
 	}
+
+	rw := lpsc.peerPool[pss.PingTopic][rpubkey]
+	lpsc.RemovePssPeer(rpubkey, pss.PingProtocol)
+	if err := rw.WriteMsg(p2p.Msg{
+		Size:    3,
+		Payload: bytes.NewReader([]byte("foo")),
+	}); err == nil {
```

Comment:
Just a suggestion to check for the explicit `connection close` error. This error could even have its own variable.